MUXSDKStats/MUXSDKStatsTests/MUXSDKStatsTests.m

```diff
@@ -8,6 +8,7 @@
 
 #import <XCTest/XCTest.h>
 #import "MUXSDKStats.h"
+#import "MUXSDKCore+Mock.h"
```

Comment:
Is this MUXSDKCore+Mock.h file missing?  I'm not seeing where `swizzleDispatchEvents`, `resetCapturedEvents` is coming from. 

I see what you're going for, this will enable us to have much better test coverage. 👍